### PR TITLE
Cleaned up text in pyplot_tutorial.rst

### PR DESCRIPTION
**Pull request by gcallah**
Only minor changes to grammar, etc.

### CODE REVIEW

File: doc/users/pyplot_tutorial.rst [hunk 1]
```diff
@@ -7,10 +7,11 @@ Pyplot tutorial
 :mod:`matplotlib.pyplot` is a collection of command style functions
 that make matplotlib  work like MATLAB.
 Each ``pyplot`` function makes
-some change to a figure: e.g., create a figure, create a plotting area
-in a figure, plot some lines in a plotting area, decorate the plot
-with labels, etc....  :mod:`matplotlib.pyplot` is stateful, in that it
-keeps track of the current figure and plotting area, and the plotting
+some change to a figure: e.g., creates a figure, creates a plotting area
+in a figure, plots some lines in a plotting area, decorates the plot
+with labels, etc.  In :mod:`matplotlib.pyplot` various states are preserved
```
Comment (tacaswell) [inline doc/users/pyplot_tutorial.rst]: I would be a bit more explicit about what state is tracked which is only the current figure and axes (there is also a `gci`, but it should not be used).

Comment (efiring) [inline doc/users/pyplot_tutorial.rst]: But the saved current image *is* used by `set_cmap`, `clim`, and `colorbar`, so it is an important part of the state.

File: doc/users/pyplot_tutorial.rst [hunk 2]
```diff
@@ -73,10 +74,11 @@ several ways to set line properties
       plt.plot(x, y, linewidth=2.0)
 
 
-* Use the setter methods of the ``Line2D`` instance.  ``plot`` returns a list
-  of lines; e.g., ``line1, line2 = plot(x1,y1,x2,y2)``.  Below I have only
-  one line so it is a list of length 1.  I use tuple unpacking in the
-  ``line, = plot(x, y, 'o')`` to get the first element of the list::
+* Use the setter methods of a ``Line2D`` instance.  ``plot`` returns a list
+  of ``Line2D`` objects; e.g., ``line1, line2 = plot(x1,y1,x2,y2)``.  In the code
```
Comment (efiring) [inline doc/users/pyplot_tutorial.rst]: We could be more consistent in formatting by adding spaces: `plot(x1, y1, x2, y2)`.

File: doc/users/pyplot_tutorial.rst [hunk 3]
```diff
@@ -204,11 +206,12 @@ as your heart desires::
 
 You can clear the current figure with :func:`~matplotlib.pyplot.clf`
 and the current axes with :func:`~matplotlib.pyplot.cla`.  If you find
-this statefulness, annoying, don't despair, this is just a thin
+it annoying that states, like which figure is current, are being maintained for
+you behind the scenes, don't despair: this is just a thin
```
Comment (efiring) [inline doc/users/pyplot_tutorial.rst]: To be fussy: the colon should be a semicolon.

**Comment by gcallah** [comment]
I have updated the document per these remarks, except: Eric, should I add the current saved image to the list of state info kept by the function-call API?

**Comment by efiring** [comment]
On 2015/09/30 4:19 PM, Gene Callahan wrote:
> I have updated the document per these remarks, except: Eric, should I
> add the current saved image to the list of state info kept by the
> function-call API?

I would say that if you list current figure and axes, then you should 
also list current image.  (Internally, the current image is tracked on a 
per-figure basis by the Figure instance.  The reason this is needed in 
addition to the current axes is that one might create 3 subplots, put an 
image-like Artist in each of the first two and a line plot in the third. 
  The current axes would then be the last subplot, but the current image 
would be in the middle subplot.)

**Comment by gcallah** [comment]
Ok, added.

**Pull request merged**